src/main.c

```diff
@@ -280,7 +283,7 @@ static void open_prefs_dialog(GSimpleAction* simple, GVariant* parameter, gpoint
 static void focal_create_main_window(GApplication* app, FocalApp* fm)
 {
 	fm->mainWindow = gtk_application_window_new(GTK_APPLICATION(app));
-	fm->weekView = week_view_new();
+	fm->weekView = week_view_new(fm->today, fm->todays_week, fm->todays_year);
```

Comment:
I'm not too fond of the idea of storing the current date *outside* of the `WeekView`. One problem is that if you leave the application open for a long time (as I think is common for a calendar application), the date will eventually be wrong and never updated. This is already handled inside the `WeekView` in order to display the red line of the current time. Perhaps this could be reused?

Reply:
i was also wondering about this, will look at how the line of current time is updated and try to suggest a reuse. thanks for your input.